huma_signals/adapters/models.py

```diff
@@ -1,16 +1,16 @@
-from typing import Any, ClassVar, Dict, List
+from typing import Any, ClassVar
 
 
 class SignalAdapterBase:
     # Signal adapters name.
     name: ClassVar[str]
     # Signals that the adapter can fetch.
-    signals: ClassVar[List[str]]
+    signals: ClassVar[list[str]]
```

Comment:
Only type annotation changes. It's recommended to use `list`, `dict` etc over `List`, `Dict` since 3.10.